src/Core/Users/UserLogin.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+namespace Meetup\Login\Core;
+
+use Meetup\Login\Core\Users\UserInterface;
+
+class UserLogin implements UserInterface {
```

Comment:
Any class that implements an interface, lets say `testInterface`, is ideally named `test`. More over this class has nothing to do wih login. Just merely is the data structure of an `user` entity. What it means is, the db will store properties exactly what the class defines.

Change: Rename `UserLogin` to `User`